CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 * Speed up encode function (now runs in ~72% less time / 3.5x improvement):
   * https://github.com/georust/polyline/pull/42
+* Add structured errors and more speedups
```

Comment:
```suggestion
* BREAKING: Return typed errors rather than String
* Performance improvements
```

Or however you want to word it - but I think it's good to call out the breaking changes.